### ISSUE TITLE
Can't apply label weight to loss function

### ISSUE THREAD

**Issue by SidneyLann** [deeplearning4j/deeplearning4j#9520]
sameDiff.loss.weightedCrossEntropyWithLogits("loss", label, out, sameDiff.var(Nd4j.create(NerUtil.labelWeight1d())));

below exception throw when run above code:

java.lang.UnsupportedOperationException: Please extend DynamicCustomOp.doDiff to support SameDiff backprop operations. Op: org.nd4j.linalg.api.ops.impl.loss.WeightedCrossEntropyLoss

No doDiff method in WeightedCrossEntropyLoss in SNAPSHOT

**Comment by SidneyLann** [comment]
Why need to implement doDiff(), it is not automatic?

**Comment by agibsonccc** [comment]
@SidneyLann  JFYI I'm removing this call. Just use softmax or sigmoid cross entropy. Those already allow you to specify weights.
